datamodel_code_generator/__main__.py

```diff
@@ -433,6 +433,15 @@ def sig_int_handler(_: int, __: Any) -> None:  # pragma: no cover
 arg_parser.add_argument(
     '--custom-file-header', help='Custom file header', type=str, default=None
 )
+
+arg_parser.add_argument(
+    '--custom-file-header-path',
+    help='Custom file header file path',
+    default=None,
+    type=str,
+)
```

Comment:
@Fokko 
I add a new option to avoid confusion.

Reply:
That works for me as well :) I was considering that as well